openvdb/Types.h

```diff
@@ -212,27 +212,93 @@ using PointDataIndex64 = PointIndex<Index64, 1>;
 ////////////////////////////////////////
 
 
-template<typename T> struct VecTraits {
+namespace internal {
+
+/// Helper metafunction used to determine if the first template parameter is a
+/// specialization of the class template given in the second template parameter
+
+template <typename T, template <typename...> class Template>
+struct SpecializationOf : std::false_type {};
```

Comment:
This is a generally useful metafunction, so I'd say there's no need to flag it as internal.  But our (admittedly loosely followed) naming convention is for predicates to read as questions.  So please rename this from `openvdb::internal::SpecializationOf` to `openvdb::IsSpecializationOf`.